packages/debloat.vm/tools/win10.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<config> 
+    <apps>
+        <!--
+        FORMAT: 
+        <app name="APP_NAME"/>
+        -->
+        <app name="Microsoft.549981C3F5F10"/>
```

Comment:
it' s not immediately clear to me what happens with the entries here
additional documentation and inline comments here would be helpful